packages/artillery-plugin-memory-inspector/index.js

```diff
@@ -0,0 +1,63 @@
+const pidusage = require('pidusage');
+const debug = require('debug')('plugin:memory-inspector');
+
+function ArtilleryPluginMemoryInspector(script, events) {
+  this.script = script;
+  this.events = events;
+
+  const inspectorConfig =
+    script.config['memory-inspector'] ||
+    script.config.plugins['memory-inspector'];
+
+  async function memoryInspectorHandler(context, ee, next) {
+    if (typeof process.env.ARTILLERY_INTROSPECT_MEMORY !== 'undefined') {
+      //https://nodejs.org/api/process.html#processmemoryusage
+      const { rss, heapUsed, heapTotal, external } = process.memoryUsage();
+      ee.emit('histogram', 'artillery_internal.memory', rss);
+      ee.emit('histogram', 'artillery_internal.external', external);
+      ee.emit('histogram', 'artillery_internal.heap_used', heapUsed);
+      ee.emit('histogram', 'artillery_internal.heap_total', heapTotal);
+    }
+
+    for (let { pid, name } of inspectorConfig) {
+      if (!pid) {
+        debug(`No pid (${pid}) or name (${name}) found. Skipping!`);
```

Comment:
The message mentions name but the name wasn't checked. Is this intentional? Do we assume this since `name` is a display name for metrics only? 

If that's the case, the `or` conjunction is misleading. 

We should also consider that `name` is optional, so it will print nothing in this message if that's the case.

Reply:
Good catch. It's an artifact of the first version of the plugin - I had `name` be mandatory, then made it optional. I think I'll just remove it from the debug message.

Reply:
Since `pid` is required, rely on that, it's enough.